apps/src/netsim/netsim.js

```diff
@@ -224,35 +224,45 @@ NetSim.prototype.init = function(config) {
    */
   this.reportingInfo_ = config.report;
 
-  config.html = page({
-    assetUrl: this.studioApp_.assetUrl,
-    data: {
-      visualization: '',
-      localeDirection: this.studioApp_.localeDirection(),
-      controls: require('./controls.html.ejs')({assetUrl: this.studioApp_.assetUrl})
-    },
-    hideRunButton: true
-  });
+  var renderCodeApp = function () {
+    return page({
+      assetUrl: this.studioApp_.assetUrl,
```

Comment:
Nice catch.  NetSim's page.html.ejs does not (though the generic one still does).